package.json

```diff
@@ -15,7 +15,8 @@
     "lint": "eslint \"./{lib,test}/**/*.js\"",
     "format": "prettier-eslint --write --list-different \"./{lib,test}/**/*.{js,json,md}\"",
     "precommit": "lint-staged",
-    "np": "np --no-publish"
+    "np": "np --no-publish",
+    "commit": "git cz"
```

Comment:
This means that `precommit` will be executed twice, the first time by `npm` before `commit` and the second time by `husky` after `commit`. See typicode/husky#99 for some workarounds.